lib/web_ui/lib/src/engine/window.dart

```diff
@@ -2,12 +2,30 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-part of engine;
+@JS()
+library window;
+
+import 'dart:async';
+import 'dart:html' as html;
+import 'dart:typed_data';
+
+import 'package:js/js.dart';
+import 'package:meta/meta.dart';
+import 'package:ui/src/engine.dart' show registerHotRestartListener;
+import 'package:ui/ui.dart' as ui;
```

Comment:
We should probably be consistent and either use relative imports or `package:` imports, but not both.

Reply:
I'm already preparing another PR that does final cleanup. I'll change to relative paths there.